packages/skip-nav/__tests__/skip-nav.test.tsx

```diff
@@ -11,23 +11,26 @@ describe("<SkipNavLink />", () => {
     });
   });
 
-  // TODO: Doesn't pass, not sure why
-  //   it("should focus the SkipNavContent on click", () => {
-  //     let { getByText } = render(<Layout />);
-  //     act(() => {
-  //       fireEvent.click(getByText("Skip Nav"));
-  //       fireEvent.keyDown(document, { key: "Tab" });
-  //       expect(getByText("Focus me")).toHaveFocus();
-  //     });
-  //   });
-  //   it("should work with a custom ID", () => {
-  //     let { getByText } = render(<Layout skipNavId="whatever" />);
-  //     act(() => {
-  //       fireEvent.click(getByText("Skip Nav"));
-  //       fireEvent.keyDown(document, { key: "Tab" });
-  //       expect(getByText("Focus me")).toHaveFocus();
-  //     });
-  //   });
+  it("should focus the SkipNavContent on click", () => {
+    let { getByText, getByTestId } = render(<Layout />);
+    const mainContent = getByTestId("main");
+    act(() => {
+      userEvent.tab();
+      fireEvent.click(getByText("Skip Nav"));
+      userEvent.tab({ focusTrap: mainContent });
+      expect(getByText("Focus me")).toHaveFocus();
+    });
+  });
+  it("should work with a custom ID", () => {
+    let { getByText, getByTestId } = render(<Layout skipNavId="whatever" />);
+    const mainContent = getByTestId("main");
+    act(() => {
+      userEvent.tab();
```

Comment:
I don't think the first tab on either test is necessary; the tests pass without it since testing-library can find the skip nav link without it showing.

Was the intention to imitate the experience of tabbing to focus the `SkipNavLink` to make it visually visible? Maybe an interesting approach to that would be to tab to show the `SkipNavLink`, then use the enter key rather than a click event to trigger.